buddyboss/buddyboss-platform
# Fix mention usernames in comments when saving

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1229 

### How to test the changes in this Pull Request:

1. In activity comment, try to reply with mentioned username which is not in users at all and post it.
2. Now try to post activity update it should be posting without delay.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
